src/www/messages/en.json

```diff
@@ -11,37 +11,37 @@
   "connecting-server-state": "Connecting...",
   "contact-page-title": "Contact us",
   "contact-view-exit-cannot-add-server": "The Outline team is not able to assist with adding a server. Please try the troubleshooting steps listed {openLink}here{closeLink} and then contact the person who gave you the access key to troubleshoot this issue.",
-  "contact-view-exit-connection": "The Outline team is not able to assist with connecting to a server. Please try the troubleshooting steps listed {openLink}here{closeLink} and then contact the person who gave you the access key to troubleshoot this issue.",
-  "contact-view-exit-no-server": "The Outline team does not distribute free or paid access keys. {openLink}Learn more about how to get an access key.{closeLink}",
+  "contact-view-exit-connection": "The Outline team is not able to assist with connecting to a server. Please try the troubleshooting steps listed {openLink}hereEND_OF_LINK and then contact the person who gave you the access key to troubleshoot this issue.",
+  "contact-view-exit-no-server": "The Outline team does not distribute free or paid access keys. {openLink}Learn more about how to get an access key.END_OF_LINK",
```

Comment:
Is this `END_OF_LINK` change expected? I think we will update `en.json` files manually?

Reply:
Good catch! Looks like I messed this up in the [source](https://github.com/Jigsaw-Code/outline-apps/blob/a0b0ac1f44e287fa2a2846993bc4f24289086036/resources/original_messages.json#L80). I'll send a PR to fix.